includes/xquery.inc

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/**
+ * Helper class for getting the query implementation(s).
+ */
+class IslandoraXquery {
+
+  /**
+   * Get the query object for the configured implementation.
+   *
+   * @param string $context
+   *   Filepath to pass as the query context.
+   * @param string $output
+   *   Filepath to receive the output.
+   * @param string $query
+   *   Filepath from which to fetch the query.
+   */
+  public static function queryObject($context, $output, $query) {
+    $class = static::getImplementation()[1];
+    return call_user_func_array(array($class, 'createBasicQuery'), func_get_args());
+  }
+
+  /**
+   * List the implementations.
+   *
+   * @return array
+   *   A mapping of machine names to class names implementing
+   *   IslandoraXQueryQueryInterface.
+   */
+  public static function getImplementations() {
+    module_load_include('inc', 'islandora_xquery', 'includes/zorba');
+    return array(
+      'zorba' => \Islandora\Zorba\Options::class,
```

Comment:
33 | ERROR | [x] Namespaced classes/interfaces/traits should be
    |       |     referenced with use statements